src/DaemonExec.ts

```diff
@@ -0,0 +1,244 @@
+import fs = require('fs')
+//import path = require('path')
+import Docker = require('dockerode')
+//needs docker to already be running
+const docker = new Docker()
+
+const listImages = async () => {
```

Comment:
Don't use arrow functions at the top level. They make it optional to specify the return type, but it's important for library functions to have a clear type signature so that they're self-documenting.